app/registries/foreman/settings/general.rb

```diff
@@ -51,6 +51,11 @@
       description: N_("Whether or not to show a menu to access experimental lab features (requires reload of page)"),
       default: false,
       full_name: N_('Show Experimental Labs'))
+    setting('new_hosts_pages',
```

Comment:
```suggestion
    setting('new_hosts_page',
```

This setting only affects one page (All Hosts), so it might be confusing to name it `pages`.